src/plugin.json

```
@@ -12,7 +12,12 @@
     },
     "description": "Simplifies data presentation with intuitive table layouts",
     "keywords": ["table"],
-    "links": [],
+    "links": [
+      {
+        "name": "Documentation",
+        "url": "https://grafana.com/docs/plugins/volkovlabs-table-panel/"
+      },
```

webpack build seems to fail because of this

```suggestion
      }
```